src/components/MenuPanel/MenuPanel.js

```diff
@@ -145,6 +146,15 @@ function MenuPanel({
     ...(showConsole ? [APP_CONSOLE] : []),
   ]
 
+  // Automatically open the menu if a system app instance is active
+  useEffect(() => {
+    const isSystemAppActive =
+      systemApps.some(systemApp => systemApp.appId === activeInstanceId) ||
+      systemAppsOpenedState.isOpen()
+    setSystemAppsOpened(isSystemAppActive)
+    setSystemAppsToggled(true)
+  }, [systemApps, activeInstanceId, systemAppsToggled, systemAppsOpened])
```

Comment:
I don't think `systemAppsToggled` and `systemAppsOpened` are necessary in the dependencies list?